src/components/select/index.js

```diff
@@ -0,0 +1,17 @@
+import { useCallback } from 'react';
+import classes from './styles.module.css';
+import { selectOptions } from '../utils/constants';
+
+const Select = props => {
+    const { handleValidityChange } = props;
+    const handleChange= useCallback( e => handleValidityChange(e.target.value));
```

Comment:
formatting, and useCallback missing dependency array

Reply:
Done